src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/utils/formComponentUtils.ts

```diff
@@ -3,7 +3,7 @@ import { ILayoutComponent, ILayoutGroup, ISelectionComponentProps } from 'src/fe
 import { IDataModelBindings, IComponentValidations, ITextResource, ITextResourceBindings, IOption, IOptions, IValidations } from 'src/types';
 
 export const isSimpleComponent = (dataModelBindings: any, type: string): boolean => {
-  const simpleBinding = dataModelBindings.simpleBinding;
+  const simpleBinding = dataModelBindings ? dataModelBindings.simpleBinding : false;
```

Comment:
Can be simplified to
```suggestion
  const simpleBinding = dataModelBindings?.simpleBinding;
```
Or even just skipping the variable assignment completely.

Reply:
Simplified